src/classes/Integration.cls

```diff
@@ -1,21 +1,22 @@
 global class Integration{
```

Comment:
Class names should be less generic, something like FinancialAdvisorsIntegration would describe what this class does better.